Userland/Libraries/LibCrypto/Hash/SHA1.cpp

```diff
@@ -132,7 +132,7 @@ SHA1::DigestType SHA1::peek()
 
     transform(m_data_buffer);
 
-    for (size_t i = 0; i < 4; ++i) {
+    for (i = 0; i < 4; ++i) {
```

Comment:
uhh...why?

Reply:
`i` was already declared in this type so this either added nothing, or it was a mistake to name the other variable `i` in the first place. I assumed the former and removed the type. Feel free to go the other way ;-)